pgjdbc/src/main/java/org/postgresql/PGArraySupport.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2004, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql;
+
+import java.sql.Array;
+import java.sql.SQLException;
+
+/**
+ * Defines public PostgreSQL extensions to create {@link Array} objects.
+ */
+public interface PGArraySupport {
```

Comment:
This is a little bit closer to Connection if comparing with other extensions, as there is `java.sql.Connection.createArrayOf()` already.